client/src/components/dashboard/AddInventory.js

```diff
@@ -0,0 +1,116 @@
+import React, {useState} from 'react';
+import Axios from 'axios';
+import Modal from 'react-modal';
+
+
+const AddInventory = () => {
+    /******/
+  const [inventoryName, setInventoryName] = useState('');
+  const [inventoryAccess, setInventoryAccess] = useState('');
+  const [inventoryDate, setInventoryDate] = useState('');
+  const [inventoryCount, setInventoryCount] = useState('');
+  const [inventoryItems, setInventoryItems] = useState('');
+  const [inventoryOwner, setInventoryOwner] = useState('');
+  const [inventoryHistory, setInventoryHistory] = useState('');
+  const [inventoryDescription, setInventoryDescription] = useState('');
```

Comment:
This is good for now, glad that you have state hooks working, but later we can make this an object with all of these properties